src/ui/RunEditor.tsx

```diff
@@ -797,7 +792,13 @@ export class RunEditor extends React.Component<Props, State> {
     private renderLeaderboard(category: Option<Category>): JSX.Element {
         const leaderboard = getLeaderboard(this.state.editor.game, this.state.editor.category);
         if (leaderboard === undefined) {
-            return <div />;
+            return <div className="run-editor-tab">
+                {this.getPlaceholderTabContent(
+                    <div className="leaderboard-spinner">
+                        <div className="fa fa-spinner fa-spin" />
+                    </div>,
+                )}
```

Comment:
We can get here? I thought we are not showing the tab anymore when there's no leaderboard?

Reply:
We don't show the leaderboard until the game is loaded, but there's still time after that where all the runs need to be loaded. I can change our logic to just hide the tab until that happens rather than showing a spinner.

Reply:
Yeah I'd prefer that.

Reply:
Though a lot of the loading and co. will change once this is happening in lsc I guess.

Reply:
We can worry about that when it happens.